frontend/app/vite.config.ts

```diff
@@ -61,6 +60,7 @@ export default defineConfig({
     },
     setupFiles: ['tests/unit/setup-files/setup.ts'],
     coverage: {
+      provider: 'istanbul',
```

Comment:
why not c8? also maybe it makes sense to split vitest + vite config?

Reply:
Sorry I saw the wrong thing, Istanbul is for e2e.